aiogram/dispatcher/dispatcher.py

```diff
@@ -276,7 +279,7 @@ def stop_polling(self):
 
         :return:
         """
-        if self._polling:
+        if hasattr(self, '_polling') and self._polling:
```

Comment:
If exception occurs before `self._polling` assign - here will be raised another one exception.
That's why I add `hasattr()` check.